.rultor.yml

```diff
@@ -74,7 +74,7 @@ release:
     git push -f heroku $(git symbolic-ref --short HEAD):master
     git reset HEAD~1
     curl -f --connect-timeout 15 --retry 5 --retry-delay 30 http://www.netbout.com
-    #mvn clean install site-deploy -Psite -P!coveralls --errors --settings ../settings.xml --batch-mode
+    mvn clean install site-deploy -Psite -P!coveralls --errors --settings ../settings.xml --batch-mode
```

Comment:
@dmzaytsev I have to remove whole profile direction or just change to -Pcoveralls?